src/RedisCache/RedisCache/help/Set-AzureRedisPrivateEndpointConnectionStatus.md

```diff
@@ -0,0 +1,159 @@
+﻿
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.RedisCache.dll-Help.xml
+Module Name: Az.RedisCache
+online version: [Set-AzRedisPrivateEndpointConnectionStatus](./Set-AzRedisPrivateEndpointConnectionStatus)
+schema: 2.0.0
+---
+
+# Set-AzRedisPrivateEndpointConnectionStatus
+
+## SYNOPSIS
+Reject Private Endpoint Connection Status from a Redis Cache.
```

Comment:
We can say like "Set Private Endpoint Connection Status from a Redis Cache."
In the later part, under parameters, we can specify the allowed values.